README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+# Biography: Madison Martin
+My name is Madison Martin, 
+*but I also go by Maddie as well*.
+I will respond to either name, I'm really not that picky. 
+ 
+I am in the MAPSS program and am pursuing the certificate in Computational Social Science.
+My main research interests are stigma, prejudice, and discrimination in academic and workplace contexts.
+Through this class, and over the course of the MAPSS program, I am hoping to:
+
+1. be introduced to the R programming language and gain coding skills 
+
+2. find new tools to improve my efforts in pyschological research, including my master's thesis project 
+
+3. expand my conception and understanding of the role of data in social science 
+
+![Chinese New Year celebration in LA earleir this year](https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/52328253_2646248942057291_7498796436759248896_o.jpg?_nc_cat=106&_nc_oc=AQnPPaILyeCG2zs9lAAXRqh9dV3H_9-J6VoGoggEtUMUUt5Ys9o5wc7Z_uaoNf7DO7g&_nc_ht=scontent-ort2-2.xx&oh=71ec929b4db32cbf0d5025dc184a68a5&oe=5E394BD2)
```

Comment:
The image was embedded correctly

Reply:
Yes, this is the correct way to embed images using a URL